### DISCUSSION TITLE
sent command Alexa/ReportState in Logbook

### DISCUSSION THREAD

**Discussion by serlinGi** [alandtse/alexa_media_player#1362] [Q&A] [answered]
hi all ,
I was wondering how to exclude from the HA-logbook the many report messages automatically generated by Alexa;  
there are many occasions in which the complete list of all devices is re-proposed and I find it quite cumbersome and not very useful. 
![Screenshot_20210827-203919_Home Assistant](https://user-images.githubusercontent.com/63103970/131174252-1dcc75a9-7c1b-4880-8f6a-cd6443c7c13c.jpg)

 Do you have any suggestions?

Thx

**Comment by CCOSTAN** [comment]
You can use the LogBook in HA to exclude the messages. 

https://www.home-assistant.io/integrations/logbook/#exclude

**Comment by alandtse** [comment, marked as answer]
This is also not related to this component at all. You're pointing to the official integration.
